scripts/up.sh

```diff
@@ -1,79 +1,54 @@
 #!/usr/bin/env bash
-#
-# Bring up the SDK test environment.
 
-set -e
+set -ae
+START=$(date "+%s")
 
-rootdir=`dirname $0`
-pushd $rootdir/.. > /dev/null
 
-# Defaults
-TYPE_OVERRIDE=""
-DAEMON_FLAG="-d"
-SKIP_BUILD=0
-SHOW_HELP=0
-ENV_FILE=".up-env"
-PARALLEL="--parallel"
+ENV_FILE=".env"
+echo "up.sh: sourcing environment vars from-->$ENV_FILE"
+source "$ENV_FILE"
```

Comment:
@tzaffi Github won't allow me to add a _suggestion_ here.  Have you considered narrowing the scope of `set -a` to be the following?

```
set -a
source "$ENV_FILE"
set +a
```

* As shown, the PR casts a wide net.  It exports _all_ variables in `up.sh`.  From my review, it seems only the highlighted line must be exported.
* I tried the suggestion locally + removing `set -a` above and sandbox starts.

Reply:
I'll double check.